fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -58,7 +59,7 @@ object FAPI {
       )
       collectionConfigs = collectionConfigJsons.map(CollectionConfig.fromCollectionJson)
     } yield {
-      (collectionIds, collectionsJsons, collectionConfigs).zipped.toList.map { case (collectionId, collectionJson, collectionConfig) =>
+      collectionIds.lazyZip(collectionsJsons).lazyZip(collectionConfigs).toList.map { case (collectionId, collectionJson, collectionConfig) =>
```

Comment:
Just want to check, would it be possible to use `scala-collection-compat` rather than building your own shim?

It looks like `ws lazyZip xs lazyZip ys` [is supported](https://github.com/scala/scala-collection-compat/blob/b26d836a629654c3117c6350df196bf025f2b7c7/compat/src/test/scala/test/scala/collection/LazyZipTest.scala#L20-L27) by [`scala-collection-compat`](https://github.com/scala/scala-collection-compat)?

Reply:
Amazing, I looked for this but only found https://github.com/giabao/scala-collection-compat-lazyzip/ which didn't look maintained.